SConstruct

```diff
@@ -17,6 +17,13 @@ env.EnsurePythonVersion(3, 5)
 # Make sure the current SCons version is at least v3.1.0; newer versions are allowed.
 env.EnsureSConsVersion(3, 1, 0)
 
+try:
```

Comment:
Is this move still necessary? I'm just asking because on my machine with scons 4.4 both produce identical compile_commands.json files.

Reply:
The scons documentation suggests putting it at the top following instantiation of env.  I can close it if you want but I'll likely keep it in my fork.